src/main/java/seedu/todo/ui/components/TagList.java

```diff
@@ -45,15 +49,19 @@ public String getFxmlPath() {
 
     @Override
     public void componentDidMount() {
-        titleText.setText("Tags (" + tags.size() + ")");
+        titleText.setText(formatTagSize(tags.size()));
 
         // Load TagListLinks
         loadLinks();
 
         // Load TagListItems
         loadTags();
     }
-
+    
+    private String formatTagSize(int size) {
+        return TAG_LABEL + SPACE + LEFT_BRACKET + size + RIGHT_BRACKET;
```

Comment:
Should be using `String.format` for this, so that brackets don't need to be extracted into this human-unreadable format.